bench-results/swe-20260416-221127-ollama-jina-embeddings-v2-base-code/detail-report.md

```
@@ -0,0 +1,660 @@
+ 			"integrity": "sha512-RdJUflcE3cUzKiMqQgsCu06FPu9UdIJO0beYbPhHN4k6apgJtifcoCtT9bcxOpYBtpD2kCM6Sbzg4CausW/PKQ==",
+-			"license": "MIT"
++			"license": "MIT",
++			"peer": true
+ 		},
+ 		"node_modules/js-yaml": {
+ 			"version": "4.1.0",
+@@ -9561,7 +9557,6 @@
+ 			"integrity": "sha512-cJW4Xd/G3v5ovXtJJ52MAOclqeac9S/aGGgRzLabuF8TnIb6xHvMzKIa6JmrRzUkeXJgfL1MhukP0NK6l39h3A==",
+ 			"devOptional": true,
+ 			"license": "Apache-2.0",
+-			"peer": true,
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)